### ISSUE TITLE
`va-additional-info` max height causes overflow

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#34991]
## Description

When the content inside a `va-additional-info` exceeds 800px in height, the overflow overlaps the content below the component

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/34878

### Reproduce it as follows:
* Log into an HLR test user - https://dsva.slack.com/archives/C5AGLBNRK/p1601297745026400?thread_ts=1601069567.024300&cid=C5AGLBNRK 
* Open https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/
* Get to https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/contestable-issues page
* Open the "Don’t see the issue you’re looking for?" additional info block
* Zoom the browser to 175% or more
* Scroll to the bottom and note it overflowing (see screenshot)

<details><summary>Content overflow</summary>

![HLR eligible issues page additional info overflowing at zoom level 175%](https://user-images.githubusercontent.com/136959/148604863-cefa52da-1809-4f95-9a30-248927079e4e.png)</details>

The `open` class has the `max-height` set to `800px` and does limit the size.

<details><summary>'open' class & inspection of element</summary>

![Screen Shot 2022-01-07 at 2 32 48 PM](https://user-images.githubusercontent.com/136959/148604816-7397ac47-adc9-403c-b14f-341667ad8fbe.png)
![Screen Shot 2022-01-07 at 2 27 24 PM](https://user-images.githubusercontent.com/136959/148604817-01c5e45d-863b-41ce-a19d-2a783b64ee58.png)</details>

### Possible solutions

- Remove the `max-height`, but I don't know if that would effect the animation
- Remove the animation & max-height
- Set the `max-height` to something outrageous
- Something else...

## Tasks
- [ ] Investigate & implement fix for overflow
- [ ] Test component at high zoom levels, up to 400%

## Definition of done
- [ ] All tasks complete
- [ ] All tests passing
